go/types/encoding_test.go

```diff
@@ -168,7 +168,7 @@ func TestRoundTrips(t *testing.T) {
 		f = math.Copysign(f, -1)
 		assertRoundTrips(Number(f))
 	}
-	floatTest := []float64{1.01, 1.001, 1.0001, 1.00001, 1.000001, 100.01, 1000.000001}
+	floatTest := []float64{1.01, 1.001, 1.0001, 1.00001, 1.000001, 100.01, 1000.000001, 122.411912027329, 0.42}
```

Comment:
I'll add them in both.

The values in these slices are negated so I need to add them outside the slices.